exporter/awsxrayexporter/awsxray_test.go

```diff
@@ -38,6 +38,17 @@ func TestTraceExport(t *testing.T) {
 	assert.Nil(t, err)
 }
 
+func BenchmarkForTraceExporter(b *testing.B) {
```

Comment:
Tests and benchmarks are fairly different in objective, would generally recommend separating PRs for them. But it's ok now for this one.